src/schema/objects/columns.yaml

```diff
@@ -339,7 +339,7 @@ onset:
     Onset (in seconds) of the event, measured from the beginning of the acquisition
     of the first data point stored in the corresponding task data file.
     Negative onsets are allowed, to account for events that occur prior to the first
-    stored data point.
+    stored data point. A `n/a` value indicates the onset time is unknown or unavailable.
```

Comment:
I would move this to the bottom to avoid breaking up discussion of negatives. And maybe "A value of `n/a`..." to get around the a/an weirdness with consonants like n (I would use "an n/a").

Reply:
moved actually at the end of the whole section

Reply:
Thanks for addressing this.  I think `n/a` is fine.  The description for `onset` now seems to be parallel to the description for `duration`.